pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "pkgconfig", "Cython!=0.27"]
+build-backend = "setuptools.build_meta"
```

Comment:
I guess this should not be backported here.

we may or may not backport this as part of backporting `black` if we decide so.